ios/RNCallKeep/RNCallKeep.m

```diff
@@ -246,6 +246,19 @@ - (void)dealloc
     }
 }
 
+RCT_EXPORT_METHOD(updateDisplay:(NSString *)uuidString displayName:(NSString *)displayName uri:(NSString *)uri)
```

Comment:
You don't need the duplicated parameter names:

`RCT_EXPORT_METHOD(updateDisplay:(NSString *)uuidString :(NSString *)displayName :(NSString *)uri)`